### DISCUSSION TITLE
'React' refers to a UMD global, but the current file is a module.

### DISCUSSION THREAD

**Discussion by CWSites** [storybookjs/storybook#18739] [Help] [answered]
* React `17.0.1`
* Storybook `6.4.13`
* Typescript `4.3.5`

React 17 no longer requires importing of `react` at the top of component files. However my Storybook files are displaying TypeScript warnings that React is referring to a global which doesn't exist and should be imported instead. 

I went through the suggestions [in this Stack Overflow question](https://stackoverflow.com/questions/64656055/react-refers-to-a-umd-global-but-the-current-file-is-a-module) and updated my `tsconfig.json` however it's still not working.

Storybook compiles, everything functions correctly, and my components themselves are fine... it's only displaying the TS error in Storybook files.

Any help would be greatly appreciated.

**Comment by elpddev** [comment, marked as answer]
It seems tsconfig is not been used by IDEs that analyze storybook path?
